server/models/courseSchema.js

```diff
@@ -60,5 +92,10 @@ module.exports.pre('save', next => {
       this.sellingText_sv = sanitize(this.sellingText_sv)
     }
   }
+  // if (this.courseDisposition_en) {
+  //   if (safeGet(() => this.courseDisposition_en)) {
+  //     this.courseDisposition_en = sanitize(this.courseDisposition_en)
```

Comment:
unnecessary comment?

Reply:
Thanks! Will look into that!

Reply:
Fixed: 48c42b6f614c6973e79250e1355caebea26500e2